content/en/docs/overview/dataplane-modes/index.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Sidecar or ambient?
+description: Learn about Istio's two dataplane modes and which you should use.
+weight: 30
+keywords: [sidecar, ambient]
+owner: istio/wg-docs-maintainers-english
+test: n/a
+---
+
+An Istio service mesh is logically split into a data plane and a control plane.
+
+The {{< gloss >}}data plane{{< /gloss >}} is the set of proxies that mediate and control all network communication between microservices. They also collect and report telemetry on all mesh traffic.
```

Comment:
These underlines do not look great. https://deploy-preview-14996--preliminary-istio.netlify.app/latest/docs/overview/dataplane-modes/

Reply:
They're standard for glossary entries?  I might agree but they're "not great" throughout if so